external-crates/move/Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
are we sure we want to update that? did you manually update cargo? normally we don't want to touch the external-crates folder.